icons/unity.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="#1976d2" d="M8 6.5 5 5l2.5-1.5V1L2 4v6.5l2-1v-3L7 8v4l-3-1.5-2 1L8 15l6-3.5-2-1L9 12V8l3-1.5v3l2 1V4L8.5 1v2.5L11 5Z"/></svg>
```

Comment:
Please make it a little lighter and increase the gaps that I marked in the photo, because when the picture is small, it feels like a box icon

<img src="https://github.com/user-attachments/assets/2331c75c-9916-40fe-8272-6a7ae9f2c1f7" width="250">